book/src/user/lightwalletd.md

```diff
@@ -158,7 +158,8 @@ Please refer to [acceptance](https://github.com/ZcashFoundation/zebra/blob/main/
 
 The final goal is to connect wallets to the lightwalletd service backed by Zebra. 
 
-For demo purposes we use [zecwallet-cli](https://github.com/adityapk00/zecwallet-light-cli).
+For demo purposes we used [zecwallet-cli](https://github.com/adityapk00/zecwallet-light-cli) with the [adityapk00/lightwalletd](https://github.com/adityapk00/lightwalletd) fork.
+[zecwallet-cli](https://github.com/adityapk00/zecwallet-light-cli) with [zcash/lightwalletd](https://github.com/zcash/lightwalletd) fork was not yet tested.
```

Comment:
###### Optional

zcash/lightwalletd is the original repo, so we don't need to refer to it as a fork.

```suggestion
We didn't test [zecwallet-cli](https://github.com/adityapk00/zecwallet-light-cli) with [zcash/lightwalletd](https://github.com/zcash/lightwalletd) yet.
```